### ISSUE TITLE
Distributed State Estimation with Deep Neural Networks for Uncertain Nonlinear Systems under Event-Triggered Communication. (arXiv:2202.01842v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#15504]
Distributed State Estimation with Deep Neural Networks for Uncertain Nonlinear Systems under Event-Triggered Communication. (arXiv:2202.01842v1 [eess.SY])<br>
https://ift.tt/qNAmaLM<br>
<p>Distributed state estimation is examined for a sensor network tasked with reconstructing a system's state through the use of a distributed and event-triggered observer. Each agent in the sensor network employs a deep neural network (DNN) to approximate the uncertain nonlinear dynamics of the system, which is trained using a multiple timescale approach. Specifically, the outer weights of each DNN are updated online using a Lyapunov-based gradient descent update law, while the inner weights and biases are trained offline using a supervised learning method and collected input-output data. The observer utilizes event-triggered communication to promote the efficient use of network resources. A nonsmooth Lyapunov analysis shows the distributed event-triggered observer has a uniformly ultimately bounded state reconstruction error. A simulation study is provided to validate the result and demonstrate the performance improvements afforded by the DNNs.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
